src/Form.tsx

```diff
@@ -64,6 +64,23 @@ const Form = (
     setFormData(newFormData);
   }
 
+  function mod(n: number, m: number) {
+    // because native JS % operator chokes on negatives
+    return ((n % m) + m) % m;
+  }
+
+  const colorShiftTooltip = () => {
+    let tip = "Start the swatch this many stitches into your color sequence."
+    const seqLength = totalColorSequenceLength(colorSequence)
+    const equiv = mod(colorShift, seqLength)
+    if (colorShift === 0 || (0 <= colorShift && colorShift < seqLength) || isNaN(equiv)) {
+      // do nothing
+    } else if (colorShift !== equiv) {
+      tip = tip + ` Shift by ${colorShift} with length ${seqLength} is equivalent to shift by ${equiv}.`
```

Comment:
I don't like using the word length here because that can be confused with stitches per row and number of rows, we don't call that value length anywhere else. Perhaps be a bit more explicit:
`because you have ${seqLength} stitches in your color sequence, shift by ${colorShift} is equivalent to shift by ${equiv}`

Reply:
Ok still thinking about wordsmithing for a minute. I wonder if we can figure out how to get $colorShift out of there so there aren't as many numbers too.

`\nSince you have ${seqLength} stitches in your color sequence, this is the same as shifting by ${equiv}`

Reply:
Would you approve a small tweak for concise text? 
`With ${seqLength} stitches in your color sequence, this is the same as shifting by ${equiv}.` 
(uses "With" in place of "Since you have")

Reply:
love it